content/en/docs/overview/singleton-service/singleton-service-apis.md

```diff
@@ -573,9 +573,160 @@ The value of 2 returns the formatted pattern data according to the provided lang
 8. machineTranslation: "true" or "false"
 9. checkTranslationStatus: "true" or "false"
 
+# **API - Get Version list API**
+This API is used to get a product name's all available versions
 
+#### **GET**/i18n/api/v2/translation/products/{productName}/versionlist
 
+**Parameters:**
+
+productName: **(required)** (String), to indicate the product name registered in Singleton; e.g. Testing
+
+**Return value:**
+
+(JSON Object) The request productName's all version list.
+
+Example:
+**{Singleton service}**/i18n/api/v2/translation/products/MULTCOMP/versionlist
+
+```
+{
+ "response":{"code":200,"message":"OK","serverTime":""},
+ "signature":"",
+ "data":{"versions":["1.0.0"],"productName":"MULTCOMP"}
+}
+```
+
+# **API - Get Mult-version Key translation**
+The API is used to get multi-version key's translation
+
+#### **GET**/i18n/api/v2/translation/products/{productName}/multiVersionKey
+
+**Parameters:**
+
+productName: **(required)** (String), to indicate the product name registered in Singleton; e.g. Testing
+
+versions: **(required)** (String), a string contains multiple translation version, separated by commas,(if versions string is ‘all’, It will get all version translation)
```

Comment:
1. typo: no , before (
2. why not use versions=[a, b, c] to represent a list, looks like separating the string by comma is not standard.

Reply:
all API use the "a,b,c" to represent a list